conans/test/functional/command/test_install_deploy.py

```diff
@@ -15,8 +17,30 @@
 def test_install_deploy():
     c = TestClient()
     c.run("new cmake_lib -d name=hello -d version=0.1")
+    c.run("create . -o *:shared=True -tf=")
+    conanfile = textwrap.dedent("""
```

Comment:
Adding an actual executable script to the test, to proof the final deploy is relocatable